kuttl/tests/helm/s1-with-smartstore/01-update-app-config.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Is this yaml invoked somewhere? I see that `s1-config.yaml` is invoked but don't see `update-app-config.yaml`, `scaleup-s1.yaml` being invoked. I might be missing something here.

Reply:
removed scaleup values yaml file as its not required.